### PR TITLE
bump jupytext version to 1.14.0

### LINKED ISSUE #213: Upgrade Jupytext in dev-environment
There seems to be an issue with our tagged Jupytext version related to this issue: https://github.com/mwouts/jupytext/issues/964

Issue: When launching `jupyter lab` having the `jupytext=1.13.8` in the activate environment, the notebooks from the docs folder having `.jupytext.toml` file present cannot be used.

### PR DESCRIPTION
**Pull request by christophfink**
closes #213

### CODE REVIEW

File: docs/md/basic-usage.md [hunk 1]
```diff
@@ -17,27 +17,27 @@ jupyter:
...
+It provides a simple and friendly interface to R<sup>5</sup> (*the Rapid Realistic Routing on Real-world and Reimagined networks*) which is a [routing engine](https://github.com/conveyal/r5) developed by [Conveyal](https://conveyal.com/). `R5py` is designed to interact with [GeoPandas](https://geopandas.org) GeoDataFrames, and it is inspired by [r5r](https://ipeagit.github.io/r5r) which is a similar wrapper developed for R. `R5py` exposes some of R5’s functionality via its [Python API](reference.html), in a syntax similar to r5r’s. At the time of this writing, only the computation of travel time matrices has been fully implemented. Over time, `r5py` will be expanded to incorporate other functionalities from R5. 
 
 <!-- #region -->
 ## Data requirements
 
 ### Data for creating a routable network
 
-When calculating travel times with `r5py`, you typically need a couple of datasets:
+When calculating travel times with `r5py`, you typically need a couple of datasets:
```
Comment (HTenkanen) [inline docs/md/basic-usage.md:27]: Hmm, there are many tiny edits here? Did they come as a result of some automation, or are these done on purpose?

Comment (HTenkanen) [inline docs/md/basic-usage.md:27]: There are many tiny edits here. Are they here on purpose?

**Review by HTenkanen — APPROVED** [review]
Looks good, but are the tiny Notebook changes there on purpose?

File: docs/md/basic-usage.md (hunk 1 above)
Comment (christophfink) [reply docs/md/basic-usage.md:27]: I guess that’s an example of the spam between versions you described. It’s always only the trailing spaces

Comment (christophfink) [reply docs/md/basic-usage.md:27]: I guess differences between jupytext versions

Comment (HTenkanen) [reply docs/md/basic-usage.md:27]: Yup, probably so. Well I guess now as we have locked the version of jupytext, there shouldn't be similar changes in the future. Hence, I think these are okay.

**Comment by HTenkanen** [comment]
@christophfink For me, this is good to go 👍🏻

**Pull request merged**